app/javascript/packs/homePageFeed.jsx

```diff
@@ -101,6 +105,7 @@ export const renderFeed = (timeFrame) => {
                 pinned={true}
                 feedStyle={feedStyle}
                 isBookmarked={bookmarkedFeedItems.has(pinnedArticle.id)}
+                saveable={pinnedArticle.user_id != currentUserId}
```

Comment:
```suggestion
                saveable={pinnedArticle.user_id !== currentUser?.id}
```

(blocking) - we should add the extra `=` the suggestion of optional chaining would allow you to skip line 93 as well , but it's just a suggestion

(same with further instances below)